src/routes/login/index.js

```diff
@@ -27,6 +67,9 @@ class Login extends Component {
   }
 
   render() {
+    const url =
```

Comment:
This constant does not need to be defined within render

Reply:
Given I only use it once, I removed it altogether and added the url directly into the component that needed it.